include/iora/iora.hpp

```diff
@@ -1117,8 +1117,9 @@ class IoraService : private core::PluginManager
     _webhookServer->setPort(port);
 
     // TLS
-    bool hasTls = _config.server.tls.certFile.has_value() ||
-                  _config.server.tls.keyFile.has_value() || _config.server.tls.caFile.has_value() ||
+    bool hasTls = _config.server.tls.certFile.has_value() &&
+                  _config.server.tls.keyFile.has_value() &&
+                  _config.server.tls.caFile.has_value() &&
```

Comment:
If require client cert is false, then hasTls will be false. Require client cert is basically a flag that indicates to the server that it needs to valdiate that the client connecting to it has a cert.